scripts/package-typescript-kiota-dependecies.ps1

```diff
@@ -0,0 +1,63 @@
+$CWD = Get-Location
+
+function Invoke-NpmPack(){
+    npm run clean
+    npm install
+    npm run build
+    npm pack
+}
+
+
+Write-Output "Packaging typescript kiota-abstractions" -ForegroundColor Green
```

Comment:
is this because kiota dependencies are not publicly available?

Reply:
Yes, the dependencies are not yet publicly released but are required

Reply:
why can you use the private feed with a token then?

Reply:
We don't have these dependencies published at all, not even on a private repo

Reply:
yes we do https://github.com/orgs/microsoft/packages?repo_name=kiota

Reply:
see the getting started docs https://microsoft.github.io/kiota/get-started/typescript.html#adding-dependencies

Reply:
I believe this is now done, resolving.